src/Controls/samples/Controls.Sample.Sandbox/MapPage.cs

```diff
@@ -0,0 +1,320 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using Microsoft.Maui;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Controls.Maps;
+using Microsoft.Maui.Devices.Sensors;
+using Microsoft.Maui.Graphics;
+using Microsoft.Maui.Maps;
+
+namespace Maui.Controls.Sample
+{
+
+	public class MapElementsPage : ContentPage
+	{
+		public MapElementsPage()
+		{
+			var grid = new Grid();
+			grid.RowDefinitions.Add(new RowDefinition());
+
+			var myhouse = new Location(47.6368678, -122.137305);
+
+			//var center = new 
+			var map = new Microsoft.Maui.Controls.Maps.Map(new MapSpan(myhouse, 0.1, 0.1));
+			grid.Children.Add(map);
+
+			// add the polygon to the map's MapElements collection
+			map.MapElements.Add(AddPolygon());
+
+			// add the polyline to the map's MapElements collection
+			map.MapElements.Add(AddPolyline());
+			//UpdatePolyLine(polyline);
+
+			// add the Circle to the map's MapElements collection
+			map.MapElements.Add(AddCircle());
+
+			Content = grid;
+		}
+
+		private static Polygon AddPolygon()
+		{
+			return new Polygon
+			{
+				StrokeWidth = 8,
+				StrokeColor = Colors.Blue,
+				FillColor = Colors.Blue.WithAlpha(0.6f),
+				Geopath =
+						{
+							new Location(47.6368678, -122.137305),
+							new Location(47.6368894, -122.134655),
+							new Location(47.6359424, -122.134655),
+							new Location(47.6359496, -122.1325521),
+							new Location(47.6424124, -122.1325199),
+							new Location(47.642463,  -122.1338932),
+							new Location(47.6406414, -122.1344833),
+							new Location(47.6384943, -122.1361248),
+							new Location(47.6372943, -122.1376912)
+						}
+			};
+		}
+
+		static Polyline AddPolyline()
+		{
+			return new Polyline
+			{
+				StrokeColor = Colors.Pink,
+				StrokeWidth = 12,
+				Geopath =
+						{
+							new Location(47.6381401, -122.1317367),
+							new Location(47.6381473, -122.1350841),
+							new Location(47.6382847, -122.1353094),
+							new Location(47.6384582, -122.1354703),
+							new Location(47.6401136, -122.1360819),
+							new Location(47.6403883, -122.1364681),
+							new Location(47.6407426, -122.1377019),
+							new Location(47.6412558, -122.1404056),
+							new Location(47.6414148, -122.1418647),
+							new Location(47.6414654, -122.1432702)
+						}
+			};
+		}
+
+		static Circle AddCircle()
+		{
+			return new Circle
+			{
+				Center = new Location(47.6381401, -122.1317367),
+				Radius = new Distance(250),
+				StrokeColor = Colors.Red,
+				StrokeWidth = 8,
+				FillColor = Colors.Red.WithAlpha(0.6f)
+			};
+		}
+
+		[Obsolete]
```

Comment:
What's the Obsolete for here? Isn't this just a sample page?

Reply:
Yeah , i will fix it